### DISCUSSION TITLE
Subpixel smoothing for dispersive materials

### DISCUSSION THREAD

**Discussion by smartalecH** [flexcompute/tidy3d#915] [Q&A]
I saw a cool snippet in one of the release notes:

https://github.com/flexcompute/tidy3d/blob/10ca408fcfbbd2c809e2490e7e823102681c0b7e/CHANGELOG.md?plain=1#L689

Does this mean that Tidy3D supports second-order accurate, subpixel smoothing for arbitrary materials (e.g. dispersive)? Or is this implying that the material lorentzians are simply being averaged (i.e. first-order accurate)?

Either way, very cool!

**Comment by momchil-flex** [comment]
I can't disclose what we're doing exactly, but it's not second-order accurate unfortunately. Also we don't support subpixel for metals right now as it seems quite nontrivial to come up with something that makes sense (and is also stable...). Would be interested if you have suggestions!

**Reply by smartalecH** [reply]
Thanks @momchil-flex! I agree it's a tough problem. Unfortunately I don't have any solutions either... but I appreciate you sharing!

**Comment by momchil-flex** [comment]
By makes sense I mean actually improves accuracy as opposed to randomly changing the result...
